cosmos/sql/tools/java/ycsb/chaos/network-faults/delay/recipes/read-patch-delay/azuredeploy.json

```diff
@@ -6,7 +6,7 @@
             "type": "string",
             "defaultValue": "Chaos",
```

Comment:
It should also be all lower case, if not the storage container will not be created. Let's change "Chaos" to "chaos".

Reply:
Since it was a quick fix, fixed all projectNames everywhere